src/interpreter_funcs/voxel_metaballs.rs

```diff
@@ -0,0 +1,303 @@
+                separate mesh shells: one for outer surface, the other for inner surface of \
+                hollow watertight mesh.",
+                refinement: ParamRefinement::Boolean(BooleanParamRefinement {
+                    default_value: true,
+                }),
+                optional: false,
+            },
+            ParamInfo {
+                name: "Distance Multiplier",
+                description: "Defines the speed of volume falloff.\n\
+                \n\
+                Lowe values mean smoother and larger results.",
+                refinement: ParamRefinement::Float(FloatParamRefinement {
+                    default_value: Some(0.5),
+                    min_value: Some(0.0001),
+                    max_value: Some(1.0),
+                }),
+                optional: false,
+            },
+            ParamInfo {
+                name: "Volume range",
+                description: "Max value 2 = fill volumes",
+                refinement: ParamRefinement::Float2(Float2ParamRefinement {
+                    min_value: Some(0.0),
+                    max_value: Some(100.0),
+                    default_value_x: Some(0.5),
+                    default_value_y: Some(2.0),
+                }),
+                optional: false,
+            },
+            ParamInfo {
+                name: "Marching Cubes",
+                description: "Smoother result.\n\
+                \n\
+                If checked, the result will be smoother, otherwise it will be blocky.",
+                refinement: ParamRefinement::Boolean(BooleanParamRefinement {
+                    default_value: true,
+                }),
+                optional: false,
+            },
+            ParamInfo {
+                name: "Prevent Unsafe Settings",
+                description: "Stop computation and throw error if the calculation may be too slow.",
+                refinement: ParamRefinement::Boolean(BooleanParamRefinement {
+                    default_value: true,
+                }),
+                optional: false,
+            },
+            ParamInfo {
+                name: "Bounding Box Analysis",
+                description: "Reports basic and quick analytic information on the created mesh.",
+                refinement: ParamRefinement::Boolean(BooleanParamRefinement {
+                    default_value: true,
+                }),
+                optional: false,
+            },
+            ParamInfo {
+                name: "Detailed Mesh Analysis",
+                description: "Reports detailed analytic information on the created mesh.\n\
+                              The analysis may be slow, therefore it is by default off.",
+                refinement: ParamRefinement::Boolean(BooleanParamRefinement {
+                    default_value: false,
+                }),
+                optional: false,
+            },
+        ]
```

Comment:
The analyses have been merged into one in the master branch meanwhile.

Reply:
i have to update it here too. this was based on an older version of master.